package.json

```diff
@@ -8,6 +8,9 @@
   "scripts": {
     "install": "node-gyp configure build"
   },
+  "dependencies": {
+    "custom-error-generator": ">=7.0.0"
```

Comment:
We should also pin all version numbers. We don't want the module to break if someone else publishes a new version that we haven't tested.